### ISSUE TITLE
Magento 2.4.7-p2 admin dashboard loading all the time can't login

### ISSUE THREAD

**Issue by AngeloZou** [magento/magento2#39068]
### Preconditions and environment

I have installed Magento multiple times, including different versions of 2.4.7-p1 and 2.4.7-p2. But I have been unable to log into the admin control panel, and it keeps loading. How can i do solve it?

My server is: **Magento 2.4.7-p2, Ubuntu/24.04 LTS, PHP/8.3.6, Apache/2.4.58, Composer version/2.7.7, elasticsearch/8.11.0.**
`Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' https://123456.com https://www.123456.com". Either the 'unsafe-inline' keyword, a hash ('sha256-64anjwo8zEwzG84h9KRhJAmwlH79VidGX9belaIjH+s='), or a nonce ('nonce-...') is required to enable inline execution.`

`Refused to execute script from '<URL>' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.`

`Uncaught Error: Script error for "jquery"
https://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js:168:17)
    at HTMLScriptElement.onScriptError (require.js:1738:36)`

`require.js:168 Uncaught 
Error: Script error for "js/theme"
https://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js:168:17)
    at HTMLScriptElement.onScriptError (require.js:1738:36)`

`Uncaught Error: Script error for "mage/adminhtml/globals"
https://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js:168:17)
    at HTMLScriptElement.onScriptError (require.js:1738:36)`

`Uncaught Error: Script error for "Magento_Catalog/catalog/product"
https://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js:168:17)
    at HTMLScriptElement.onScriptError (require.js:1738:36)`

`require.js:168 Uncaught 
Error: Script error for "mage/backend/bootstrap"
https://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js:168:17)
    at HTMLScriptElement.onScriptError (require.js:1738:36)`

`Uncaught Error: Script error for "Magento_Translation/js/mage-translation-dictionary"
https://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js:168:17)
    at HTMLScriptElement.onScriptError (require.js:1738:36)`

`Uncaught (in promise) Error: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received`

`<script>
    var adminAnalyticsMetadata = {
        "secure_base_url": "http\u003A\u002F\u002Fwww.123456.com\u002F",
        "version": "2.4.7\u002Dp2",
        "product_edition": "Community",
        "user": "a6cf695973a3e9f52563a525b10e263f64eb51e360dc1f9a811cd121e4e8ea0f",
        "mode": "default",
        "store_name_default": "",
        "admin_user_created": "2024\u002D08\u002D19\u002021\u003A08\u003A41",
        "admin_user_logdate": "2024\u002D08\u002D20\u002005\u003A11\u003A13",
        "admin_user_role_name": "Administrators"
    };
    var digitalData = {
        "nonce": "Mjlkb3dxYWl2MmY3NmU5NjFwa285bTU3d3dncHd2d3I\u003D"
    };
    var cspNonce = "Mjlkb3dxYWl2MmY3NmU5NjFwa285bTU3d3dncHd2d3I\u003D";
</script>`

`The Content Security Policy (CSP) prevents cross-site scripting attacks by blocking inline execution of scripts and style sheets.
To solve this, move all inline scripts (e.g. onclick=[JS code]) and styles into external files.
⚠️ Allowing inline execution comes at the risk of script injection via injection of HTML script elements. If you absolutely must, you can allow inline script and styles by:
adding unsafe-inline as a source to the CSP header
adding the hash or nonce of the inline script to your CSP header.`

`Some resources are blocked because their origin is not listed in your site's Content Security Policy (CSP). Your site's CSP is allowlist-based, so resources must be listed in the allowlist in order to be accessed.
A site's Content Security Policy is set either via an HTTP header (recommended), or via a meta HTML tag.
To fix this issue do one of the following:
(Recommended) If you're using an allowlist for 'script-src', consider switching from an allowlist CSP to a strict CSP, because strict CSPs are more robust against XSS . See how to set a strict CSP .
Or carefully check that all of the blocked resources are trustworthy; if they are, include their sources in the CSP of your site. ⚠️Never add a source you don't trust to your site's CSP. If you don't trust the source, consider hosting resources on your own site instead.`
![Screenshot_1](https://github.com/user-attachments/assets/db3834d1-1f4c-4acf-8d7d-2b5d11d4917e)
![Screenshot_2](https://github.com/user-attachments/assets/28b11c50-d2ae-443b-a79a-9ec3d1dd258a)
![Screenshot_3](https://github.com/user-attachments/assets/3fdde9d3-1689-486b-8be8-a686ef07778d)
![Screenshot_4](https://github.com/user-attachments/assets/003cf2cb-ce29-46ac-a6b3-3a27ff0adf62)

### Steps to reproduce

1、Log in to Magento admin panel.
2、Unable to enter after logging in.
3、It keeps loading, and I can't click on any menu.

### Expected result

Log in to the backend to open any menu.

### Actual result

I can't open the menu, it keeps loading. I've tried installing different versions, and I got the same result.

### Triage and priority

- [x] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by engcom-Bravo** [comment]
Hi @AngeloZou,

Thanks for your reporting and collaboration.

We have verified the issue by installing 2.4.7-p2 and we are able to login and able to open backend menu.

<img width="1436" alt="Screenshot 2024-08-22 at 13 41 21" src="https://github.com/user-attachments/assets/1b9cab04-6dc9-47c7-b5e1-9d00c7ef4117">


Thanks

**Comment by AngeloZou** [comment]
I'm logged in, but I can't open any menus. The bugs are still the same.

**Comment by AngeloZou** [comment]
I have solved this problem.

nano .htaccess
```
<IfModule mod_headers.c>
  Header set Access-Control-Allow-Origin "*"
</IfModule>
```

`bin/magento config:set web/secure/base_url https://yoursite.com/`

**Comment by engcom-Bravo** [comment]
Hi @AngeloZou,

Thanks for your update.

As per this comment https://github.com/magento/magento2/issues/39068#issuecomment-2306716006 we are closing this issue.

Thanks.

**Comment by AngeloZou** [comment]
I have found the main reason. The default installed Elasticsearch version in this tutorial is 8.15.0. I downgraded the version to Elasticsearch 8.11.0, and the problem was solved.

`sudo apt-get remove --purge elasticsearch`
`sudo rm -rf /etc/elasticsearch/`
`sudo rm -rf /var/lib/elasticsearch/`
`wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-8.11.0-amd64.deb`
`sudo dpkg -i elasticsearch-8.11.0-amd64.deb`
`sudo nano /etc/elasticsearch/elasticsearch.yml`
yaml 
network.host: 127.0.0.1 
http.port: 9200 
xpack.security.enabled: false
`sudo systemctl start elasticsearch`
`sudo systemctl enable elasticsearch`
